src/content/enterprise/install/releases.mdx

```diff
@@ -5,6 +5,28 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 0.13.0
+TBD
+
+### Features
+- Add Access Control to [Volume Snapshots](/docs/enterprise/administration/volume-snapshots/)
+- Add link to Community [Community](https://community.okteto.com) in help menu.
+- Inject OKTETO_NAMESPACE and OKTETO_DOMAIN environment variables in every pod
+- Remove Namespace limit on licenses
+
+### Improvements
```

Comment:
Should we include the change to disable autoscrolling in the logs?

Reply:
I never understood that issue/fix. I thought it was transparent to the user, or is it a bug fix? cc @viddo